src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/InterimUploadAdditionalDocument.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class InterimUploadAdditionalDocument {
```

Comment:
Why not just use the already existing CaseDocument POJO?

Reply:
Had an issue. So I went with this approach